src/Storages/RabbitMQ/RabbitMQSource.cpp

```diff
@@ -261,7 +288,6 @@ Chunk RabbitMQSource::generateImpl()
     for (auto & column : virtual_columns)
         result_columns.push_back(std::move(column));
 
-    commit_info = current_commit_info;
```

Comment:
One more question: is `commit_info.failed_delivery_tags` are cleaned up amywhere? Isn't it a problem if we don't clean them up?

Reply:
`commit_info` is a property of Source, not Consumer, Consumer is reused for reading while the Source is one time usage, e.g. we read and then ack/nack, after ack/nack we do not use the Source anymore, so it should not matter if we clear `commit_info` or not.